pkg/venafi/firefly/connector.go

```diff
@@ -309,15 +313,22 @@ func (c *Connector) getCertificateRequest(req *certificate.Request) *certificate
 	keyAlgorithm := ""
 	switch req.KeyType {
 	case certificate.KeyTypeRSA:
-		keyAlgorithm = fmt.Sprintf("RSA_%d", req.KeyLength)
-
+		keyLength, err := GetRSASize(req.KeyLength)
```

Comment:
change the var name to `keySize` instead of `keyLength`

Reply:
keyLength is the concept used by CertificateRequest structure so I've decided to keep the var on that way

Reply:
then change the name of the function. size is the correct term to refer to the RSA key type. However if we are going to use length, then lets use it consistently everywhere

Reply:
Agree...

Reply:
Done